### ISSUE TITLE
Import layout from xkb

### ISSUE THREAD

**Issue by tobiasBora** [kmonad/kmonad#843]
TL;DR: Is it possible to import a layout from `setxkbmap`? For instance I’d like to start from the layout `setxkbmap fr bepo_afnor` on the default template ISO 100%, but manually creating all layers seem quite tedious.

Motivations: I wanted to use kmonad to remap various keys, but I’m thinking it could be a good opportunity to let kmonad deal with my whole configuration. As far as I understand there are 2 options right now:
1. Either write `_` everywhere in the configuration to let kmonad forward the keys to the system (letting ibus/setxkbmap do the job)… but then it is not possible to create really advanced mechanisms due to https://github.com/kmonad/kmonad/issues/842 that forbids me to write for instance `(tap-hold 200 _ x)`
2. Manually define a layer (bépo, qwerty, …) from scratch. This can be quite tedious, especially on such large layouts (technically, bepo defines latin/greek/… letters and many more symbols that would take me ages to write)

Am I missing an obvious way to import an existing layout?

**Comment by slotThe** [comment]
There's no real way to import a layout from another place.

> Also, it seems like there is no way to send an "absolute" symbol (for instance if I write x in kmonad, and if I type setxkbmap fr bepo_afnor, then it will actually write a y since bépo maps the x key to y…)

This is quite literally impossible, because kmonad sits "underneath" X11's localisation settings. We are sending an `x` keypress, but since you've set your keyboard to `bepo` afterwards, this is (correctly!) intercepted and translated according to that keymap. Note that `_` doesn't really mean "pass it to the system", but rather "emit whatever key is on the layer beneath us right now".

I would imagine the easiest solution is to define a QWERTY layout and then let `setxkbmap` handle the BEPO stuff
